test/suites/dev/test-precompile/test-precompile-multiple-locks.ts

```diff
@@ -0,0 +1,57 @@
+import "@moonbeam-network/api-augment";
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import {
+  GLMR,
+  MIN_GLMR_DELEGATOR,
+  alith,
+  checkBalance,
+  createRawTransfer,
+  generateKeyringPair,
+} from "@moonwall/util";
+import { createProposal } from "../../../helpers/voting.js";
+
+describeSuite({
+  id: "D2580",
+  title: "Precompiles - Locks - Voting and staking locks are not mutually exclusive",
+  foundationMethods: "dev",
+  testCases: ({ context, it }) => {
+    let proposalIndex: number;
+    const randomAccount = generateKeyringPair();
+    const randomAddress = randomAccount.address as `0x${string}`;
+    beforeAll(async function () {
+      await context.createBlock(
+        createRawTransfer(context, randomAddress, MIN_GLMR_DELEGATOR + GLMR)
+      );
+      proposalIndex = await createProposal(context);
+    });
+
+    it({
+      id: "T01",
+      title: "should be able to vote (conviction) and stake",
+      test: async function () {
+        // Vote yes on proposal with 1 GLMR
+        await context.createBlock(
+          context
+            .polkadotJs()
+            .tx.convictionVoting.vote(proposalIndex, { Standard: { Vote: 0, Amount: GLMR } })
+            .signAsync(randomAccount),
+          { allowFailures: false }
+        );
+
+        // Delegate to Alith with MIN_GLMR_DELEGATOR
+        await context.createBlock(
+          context
+            .polkadotJs()
+            .tx.parachainStaking.delegate(alith.address, MIN_GLMR_DELEGATOR, 0, 0)
+            .signAsync(randomAccount),
+          { allowFailures: false }
+        );
+
+        // assert that sum of conviction and staking locks is greater than final balance
+        // ie, that the locks are not mutually exclusive
+        const balance = await checkBalance(context, randomAddress);
+        expect(Number(balance)).to.be.lessThan(Number(MIN_GLMR_DELEGATOR + GLMR));
```

Comment:
I think this is always going to be true, given that `MIN_GLMR_DELEGATOR + GLMR` is the initial balance.
Another possible way (IIRC)) to know if the locks are not exclusive is to check that the locked balance returned by `system.account()` is equal to `max(GLMR, MIN_GLMR_DELEGATOR)`

Reply:
thanks, added